Domains/Smartlighting/LightSchedules.md

```diff
@@ -0,0 +1,286 @@
+# Light Schedules
```

Comment:
I thought this would be clear enough, enforced by the notion that these pictures do not stand by themselves, but are in most places accompanied by a comment indicating whether sunset or sunrise applies, and are in all cases placed near a time for an hour of the day that should also indicate sunrise or sunset.
With the added idea that there aren't ready made symbols for what you ask, I think it would be quite a bit of work for me to replace them all, and I wonder if the improvement would be significant enough for that.